doc/api/stream.md

```diff
@@ -2558,6 +2558,7 @@ class WriteStream extends Writable {
   constructor(filename) {
     super();
     this.filename = filename;
+    this.fd = null;
```

Comment:
Given that a file descriptor is a number, using -1 might seems a better default.